charts/observability-pipelines-worker/values.yaml

```diff
@@ -39,12 +32,20 @@ datadog:
   dataDir: "/var/lib/observability-pipelines-worker"
   # datadog.remoteConfigurationEnabled -- Whether to allow remote configuration of the worker from Datadog.
   remoteConfigurationEnabled: false
+  worker:
```

Comment:
I went with `datadog.worker.api` instead of `datadog.api` since it felt a bit clearer -- open to changing this.

Reply:
Yeah, I was gonna say that pretty much all of the options under `datadog` are "worker" related. Maybe we could remove the `worker` namespacing and call `api`, `worker_api, `local_api`, or `api_server`? That at least gives less of the impression that other options don't apply to the worker.

Reply:
Sounds good, I went with `workerAPI` in [f685357](https://github.com/DataDog/helm-charts/pull/1144/commits/f685357795bf3d08c85053917ec7ec97c3cc0f3b)